src/include/rados/librados.hpp

```diff
@@ -14,6 +14,8 @@
 #include "librados_fwd.hpp"
 #include "rados_types.hpp"
 
+#include "cls/cls_traits.h"
```

Comment:
cls_traits.h and cls_flags.h should probably be under src/include/rados and installed with the other headers by src/include/CMakeLists.txt for inclusion in packages like librados-devel and libradospp-devel